src/UI Component/Adminpanel/Activities/loanapproval.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+
+const LoanApprovalPage = () => {
+  // Dummy data for loan applications
+  const loanApplications = [
+    { id: 1, customerName: 'John Doe', loanAmount: 5000, status: 'Pending',comment: 'Blurry Bank Statement '},
+    { id: 2, customerName: 'Jane Smith', loanAmount: 8000, status: 'Pending',  comment: 'incomplete document'},
+    { id: 3, customerName: 'Bob Johnson', loanAmount: 10000, status: 'Pending', comment: 'incomplete document' },
+    // Add more entries as needed
+  ];
+
+  const [applications, setApplications] = useState(loanApplications);
+
+  const handleApproveReject = (id, action) => {
+    const updatedApplications = applications.map((app) => {
+      if (app.id === id) {
+        return { ...app, status: action };
+      }
+      return app;
+    });
+
+    setApplications(updatedApplications);
+  };
+
+  return (
+    <div className=" h-screen mx-auto mt-16">
+      <h1 className="text-xl  border border-b-blue-900 font-bold mb-6">Loan Approval</h1>
+
+      {/* Loan applications grid */}
```

Comment:
Is there a reason why you are using grid here and then using table up in the code?

Reply:
I opted for div elements to provide a more flexible styling approach for rows and columns in the structure. However, I've replaced the table element with div elements in the code.